extract_assets.py

```diff
@@ -69,7 +69,6 @@ def initializeWorker(abort, unaccounted: bool, extractedAssetsTracker: dict, man
 def main():
     parser = argparse.ArgumentParser(description="baserom asset extractor")
     parser.add_argument("-s", "--single", help="asset path relative to assets/, e.g. objects/gameplay_keep")
-    parser.add_argument("-t", "--threads", help="Number of cpu cores to extract with.")
```

Comment:
Why did you remove this?

Reply:
I copied most of the OoT script over so as to only maintain one. This had the side effect that all the `-t`-related code is gone.

Reply:
Could you add it back?

Reply:
How? I don't understand how you want it to work.

Reply:
The same way it is working currently.
Use the value of args,threads for `numCores` instead of using `cpu_count`